js/ui/components/clubhouse.js

```diff
@@ -217,6 +221,16 @@ var ClubhouseButtonManager = new Lang.Class({
         this._clubhouseWindowActor = null;
     },
 
+    startPulsingIcon: function() {
+        this._openButton.get_children()[0].add_style_pseudo_class('highlighted');
```

Comment:
I don't love that we are getting the con this way. Maybe creating a subclass of the button called `NotifiableButton` that gets a `pulsate` method is cleaner and easier to maintian.